addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -311,6 +311,11 @@ function factory(dependencies) {
     }
 
     Chatter.fields = {
+        attachmentBoxView: one2one('mail.attachment_box_view', {
+            default: insertAndReplace(),
```

Comment:
As suggested above, this should not be always set, but only inserted or cleared when we currently write true or false on `isAttachmentBoxVisible`.